src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -691,6 +686,10 @@ void NotationViewInputController::mouseReleaseEvent(QMouseEvent*)
         && !interaction->isDragStarted() && !noteInput->isNoteInputMode()) {
         interaction->clearSelection();
     }
+    if (event->button() == Qt::MouseButton::LeftButton && m_view->toLogical(event->pos()) == m_beginPoint
+        && !(event->modifiers() & Qt::KeyboardModifier::ControlModifier) && hitElement()) {
+        interaction->select({ hitElement() });
```

Comment:
On releasing the left mouse button, if you clicked on an element, and haven't moved the mouse, and aren't holding down CTRL, then the existing selection should be cleared and replaced with the clicked-on element.